main.go

```diff
@@ -337,13 +337,18 @@ func triggerBuild(conf *config, build *buildOptions) error {
 		Ref: &ref,
 		Variables: buildParameters,
 	}
-	integrationPipelinePath := "Northern.tech/Mender/mender-qa"
-	log.Info("Creating pipeline in project " + integrationPipelinePath + " with opts: " + spew.Sdump(opt) + "\n")
+
+	variablesString := ""
+	for _, variable := range opt.Variables {
+		variablesString += variable.Key + ":" + variable.Value + ", "
+    }
+	log.Infof("Creating pipeline in project %s:%s with variables: %s", integrationPipelinePath, *opt.Ref, variablesString)
```

Comment:
Confused... You just removed this variable above...?

Reply:
Ups. That was a bad commit. This will not compile

Reply:
Sorry. Fixed now.